docs/README.md

```diff
@@ -34,6 +34,55 @@ You'll need a `kubectl`-style config file to connect to your cluster.
 `$HOME/.kube/config`. You can specify an alternate kubeconfig connection file
 with `--kubeconfig`, or the env var `$KUBECONFIG`, for any command.
 
+## Kn Config
+
+To better customize Kn there is a small set of configuration parameters you are able to setup. In particular you can specify where your Kn plugins are located and how they are found. The Kn configuration file is meant to capture these limited configuration options. Let's explore this file, it's location, and the current options you are able to change with it.
+
+### Location
+
+By default the Kn config file is located in the user's home directory under `.kn` hidden subdirectory and is named `config.yaml`. So in `~/.kn/config.yaml`. It is not created for you by default. You can easily create one with:
+
+```bash
+mkdir ~/.kn
+touch ~/.kn/config.yaml
+```
+
+Once that file exists (even when empty) Kn will indicate that is being used when you invoke a command. And default values for the configurable parameters will be added (if they were not present). For instance:
+
+```bash
+./kn plugin list
+Using kn config file: /home/maximilien/.kn/config.yaml
+No plugins found.
+```
+
+To see the values in the Kn config file you can do:
+
+```bash
+cat ~/.kn/config.yaml 
+lookuppluginsinpath: false
```

Comment:
isn't it in camel case ? (should be imo)

Reply:
Not in the file. The way Viper works is to make it case insensitive--as far as I can tell. Not my first choice.

I've been meaning to open an issue on this and investigate if we can do something about it. I agree it would be better if if was case sensitive and camel cased in the config file.

Reply:
See issue #414

Reply:
Submitting this as is and when #414 is resolved I'll make necessary changes that matches resolution. Hopefully camel case.